definitions/variable/CO2.yaml

```
@@ -0,0 +1,5 @@
+# This yaml file adds needed variables for E3M´s model data upload
+
+- Capacity|CO2|Direct Air Capture:
```

Suggest to be more explicit because there may be other categories related to CO2 infrastructure (CO2 pipelines, injection into geological storage, etc.)
```suggestion
- Capacity|Carbon Capture|Direct Air Capture:
```